src/android/com/silkimen/cordovahttp/CordovaHttpPlugin.java

```
@@ -110,6 +112,19 @@ public boolean execute(String action, final JSONArray args, final CallbackContex
     }
   }
 
+  private boolean checkClientCertValidity(final JSONArray args, final CallbackContext callbackContext){
+    try {
+    String alias = args.getString(0);
+      CordovaClientCertCheck runnable = new CordovaClientCertCheck(alias, this.cordova.getActivity(), callbackContext);
+
+      cordova.getThreadPool().execute(runnable);
+    } catch (JSONException e){
```

remove catch and let it throw